### DISCUSSION TITLE
How do i find bugs/exploits?

### DISCUSSION THREAD

**Discussion by IGETboard** [3kh0/ext-remover#1351] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How do i find bugs or exploits for school and use them?

**Comment by schoolexploitkid** [comment]
1. Check chrome urls for point blank
2. Find point blank in ur block extension
3. Find code execution in adblockers and weird extensions that are allowed by ur school

**Reply by IGETboard** [reply]
How do i find point blank like what to look for?

**Reply by schoolexploitkid** [reply]
in the view source, `window.open()` or `contenteditable=true`

**Reply by akabutnicer** [reply]
i wouldn't bother looking for point blank anymore, that prototype of an 'exploit' typically involves bookmarklet access, which as of v115, is blocked regardless.

**Reply by IGETboard** [reply]
What are some other ways then?

**Reply by schoolexploitkid** [reply]
look for xss attack in block pages or extension pages

**Comment by IGETboard** [comment]
More ways to find bugs/exploits here. I need some tips

**Reply by akabutnicer** [reply, marked as answer]
it's not that easy for us to TELL YOU how to find these, it's something you will need to put your own time into. if you look in the right places (which isn't easy to disclose abstractly), then the exploit will come to you eventually. i would recommend you try and extend an already existing exploit if you aren't that experienced in such a field.

**Reply by akabutnicer** [reply]
@S-PScripts close this

**Reply by S-PScripts** [reply]
men
